core/src/main/scala/com/salesforce/op/OpWorkflowModelWriter.scala

```diff
@@ -180,19 +197,6 @@ private[op] object OpWorkflowModelReadWriteShared {
  */
 object OpWorkflowModelWriter {
 
-  /**
-   * Save [[OpWorkflowModel]] to path
-   *
-   * @param model     workflow model instance
-   * @param path      path to save the model and its stages
-   * @param overwrite should overwrite the destination
-   */
-  def save(model: OpWorkflowModel, path: String, overwrite: Boolean = true): Unit = {
```

Comment:
the scope of this PR did not require it, we should not do breaking changes such as removing public methods.

Reply:
Oh, I missed that one! good catch @gerashegalov

Reply:
@nicodv please revert this part of the change.

Reply:
@gerashegalov / @tovbinm So we are OK with adding an implicit SparkSession to `OpWorkflowModel.save()`?

Reply:
@nicodv this is not the choice we have to make. We can make it consistent by just changing `saveImpl` in `class OpWorkflowModelWriter`
```scala
  override protected def saveImpl(path: String): Unit = {
    JobGroupUtil.withJobGroup(OpStep.ModelIO) {
      sc.parallelize(Seq(toJsonString(path)), 1)
        .saveAsTextFile(OpWorkflowModelReadWriteShared.jsonPath(path), classOf[GzipCodec])
    }(sparkSession)
  }
```

Reply:
Ah, I overlooked the `saveImpl` method. New PR that reverts this: https://github.com/salesforce/TransmogrifAI/pull/470